### DISCUSSION TITLE
Issue with contribution - docs folder

### DISCUSSION THREAD

**Discussion by matthewhausman** [wevm/wagmi#1156] [General]
/Users/matthausman/test-wagmi/wagmi/docs/components/core/Providers.tsx
  10:41  error  Unable to resolve path to module 'wagmi/connectors/coinbaseWallet'  import/no-unresolved
  11:35  error  Unable to resolve path to module 'wagmi/connectors/injected'        import/no-unresolved
  12:35  error  Unable to resolve path to module 'wagmi/connectors/metaMask'        import/no-unresolved
  13:40  error  Unable to resolve path to module 'wagmi/connectors/walletConnect'   import/no-unresolved
  15:33  error  Unable to resolve path to module 'wagmi/providers/alchemy'          import/no-unresolved

for some reason these import paths are not able to be resolved and fails the precommit hooks. I have made 0 changes to the repository aside from adding code for a vue package

**Comment by tmm** [comment]
Try `pnpm dev` before committing. Possible packages aren't linked across the monorepo.

**Reply by matthewhausman** [reply]
doesn't seem to have any impact :/ (on both the linter showing errors and precommit hooks failing)

**Reply by matthewhausman** [reply]
On a clean clone there is no issue, has to be something related to the incompleteness of what I have so far

**Reply by matthewhausman** [reply]
got it :)
